addons/radios/functions/fnc_lowerHeadset.sqf

```diff
@@ -0,0 +1,43 @@
+/*
+Author: Grey
+
+Adjusts SW & LR radio volume
+
+Arguments:
+0: unit <OBJECT>
+
+Return value:
+None
+*/
+#include "script_component.hpp"
+
+private "_haveSW,_haveLR,_radioSW,_radioLR,_isLowered";
+
+_haveSW = call TFAR_fnc_haveSWRadio;
+_haveLR = call TFAR_fnc_haveLRRadio;
+
+_radioSW = call TFAR_fnc_activeSwRadio;
+_radioLR = call TFAR_fnc_activeLrRadio;
+
+_isLowered = ACE_player getVariable [QGVAR(headsetIsLowered),false];
+if(_isLowered) then{
+	if (_haveSW)then{
+		[_radioSW, 8] call TFAR_fnc_setSwVolume;
+	};
+
+	if (_haveLR)then{
+		[(call TFAR_fnc_activeLrRadio) select 0, (call TFAR_fnc_activeLrRadio) select 1, 8] call TFAR_fnc_setLrVolume;
```

Comment:
Two times a call to `TFAR_fnc_activeLrRadio`. Better would be:
```sqf
(call TFAR_fnc_activeLrRadio) params ["_valueA", "_valueB"];
[_valueA, _valueB, VOLUME_NORMAL] call TFAR_fnc_setLrVolume;
```
So two things:

1. Combine the two calls to activeLrRadio and extract the values we want by using `params`
2. Store the normal volume (`8`) into a define on the top of the file, and call it `VOLUME_NORMAL` or something along those lines.

Example of a define:
```sqf
#define VOLUME_NORMAL 8
```